src/bin/common.rs

```diff
@@ -33,6 +33,7 @@ pub struct CliOptions {
   pub override_time_base: bool,
   pub skip: usize,
   pub verbose: bool,
+  pub quiet: bool,
```

Comment:
IMO it would be cleaner to have one field for `verbosity` with a tri-state enum with states `Quiet`, `Normal`, `Verbose`.

Reply:
You are right.